### ISSUE TITLE
SLU08/13 curriculum overlap

### ISSUE THREAD

**Issue by CatarinaSilva** [LDSSA/curriculum-development#36]
## Context

Improvements to bootcamp flow

## Detailed Description

SLU08 has 

```
Using the Metrics

    Hold-out method
        Training error
        Testing error
    K-Fold
        sklearn.model_selection.cross_val_score(scoring=)
```

then SLU10 ignores these topics, even though it's the analogous unit to this one for classification.

And in SLU13 they come back with much more context and with a much deeper view over these:

```
Model Selection

    Offline evaluation
        Leave-one-out or hold-out method
            sklearn.model_selection.train_test_split
            In-sample or training error
            Out-of-sample or testing error
            Validation dataset
            Evaluating overfitting and underfitting
        K-Fold cross-validation
            sklearn.model_selection.cross_val_score
        Data leakage
    Practical considerations
        Training time
        Prediction time
```
## Possible Implementation

Remove how to use from SLU08, making it consistent with SLU10 and ensuring a better split of information.

**Comment by UrbanoFonseca** [comment]
SLU13 is pretty good for students to perceive the different metric estimation techniques, applicable to both regression and classification, and it covers all the techniques mentioned in SLU08.
:+1: Agree that we can remove the "Using the metrics" section from SLU08 but since this is taking most of the learning notebook (like 60%), we must replace with something more meaningful to the students. 

I'm in favour of replacing the "Using the metrics" by more practical examples about the metrics themselves and giving them a better intuition (e.g. what each metric measures in reality, its transposability to the real world and how to choose between the multiple options), in a similar fashion SLU10 does for introducing classification metrics.

Wdyt @CatarinaSilva ?

**Comment by UrbanoFonseca** [comment]
SLU13 is pretty good for students to perceive the different metric estimation techniques, applicable to both regression and classification, and it covers all the techniques mentioned in SLU08.
:+1: Agree that we can remove the "Using the metrics" section from SLU08 but since this is taking most of the learning notebook (like 60%), we must replace with something more meaningful to the students. 

I'm in favour of replacing the "Using the metrics" by more practical examples about the metrics themselves and giving them a better intuition (e.g. what each metric measures in reality, its transposability to the real world and how to choose between the multiple options), in a similar fashion SLU10 does for introducing classification metrics.

Wdyt @CatarinaSilva ?

**Comment by UrbanoFonseca** [comment]
cc/ @marianahenriques1

**Comment by CatarinaSilva** [comment]
Sounds good to me @UrbanoFonseca !
